.circleci/build-and-test/jobs.yml

```diff
@@ -31,7 +31,7 @@
           command: cd tdrs-frontend; npm run lint
       - run:
           name: Run Pa11y Accessibility Tests
-          command: cd tdrs-frontend; mkdir pa11y-screenshots/; npm run test:accessibility
+          command: cd tdrs-frontend; mkdir pa11y-screenshots/; export NODE_OPTIONS=--openssl-legacy-provider; npm run test:accessibility
```

Comment:
Will we need to do this anywhere else?

Reply:
long story short - you never really know, and some scripts may require you to `unset NODE_OPTIONS` as well. the test pipeline should run just about every `npm` script we have, with the exception of maybe `build:production` (some version of `build` is run for the tests). you could try running it in circle ci via ssh to test it out.